community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/index/LuceneIndexAccessor.java

```diff
@@ -172,35 +174,26 @@ private LuceneIndexUpdater( boolean inRecovery )
         }
 
         @Override
-        public void process( NodePropertyUpdate update ) throws IOException
+        public PreparedIndexUpdates prepare( Iterable<NodePropertyUpdate> updates ) throws IOException
```

Comment:
This method does the right thing in terms of returning a `PreparedIndexUpdates` that will perform the actual application of the updates upon `commit()`, why don't the other implementations of `prepare(...)` do it that way?